### ISSUE TITLE
The downloaded name is not prefixed by user id.

### ISSUE THREAD

**Issue by alan23273850** [HuiiBuh/InstagramDownloader#322]
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to URL 'https://www.instagram.com/stories/xrspace.official/'
2. Click on 'the download button on the top right corner'
3. See error: The downloaded name is 'unknown_xxx'

**Expected behaviour**
The downloaded name should be '{expected_user_id}_xxx'

**Screenshots**
<img width="300" alt="image" src="https://user-images.githubusercontent.com/10044077/186419810-c013d0c4-41c9-4abf-ade4-065f5befb6b5.png">

**Device (please complete the following information):**
- Resolution: 1080P
- Browser: Chrome 104.0.5112.102, 64-bit version
- Extension Version: 4.6.4

**Comment by dutchcube** [comment]
Also having this issue for a while now. A fix would be amazing! Thanks for the work.

**Comment by ilovescraperz** [comment]
Yup been having the same issue, including stories. For stories it's been since at least August 18th

**Comment by Allen-Kwan** [comment]
I assume it's something here, but hopefully someone can maybe make a PR or something to fix it.

`        const accountName: HTMLHeadingElement | null = document.querySelector(QuerySelectors.accountName);

        const downloadMessage: DownloadMessage = {
            imageURL: [image.src],
            accountName: accountName?.innerText || 'unknown',
            type: DownloadType.single,
        };
        await browser.runtime.sendMessage(downloadMessage);`

**Comment by alan23273850** [comment]
@Allen-Kwan You can simply do it!

**Comment by Runningrock24** [comment]
This can't be solved?

**Comment by ForAnAngel731** [comment]
This doesn't seem to have been fixed.

**Comment by Allen-Kwan** [comment]
@TomK32 - anyway you could make installation packages for Chrome/Firefox with your update?

**Comment by TomK32** [comment]
@Allen-Kwan I'm afraid it's @HuiiBuh who has to upload those packages.

**Comment by alan23273850** [comment]
I've found that https://chrome.google.com/webstore/detail/story-saver/mafcolokinicfdmlidhaebadidhdehpk is a useful alternative to this one. Have fun with it!
